scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1604,32 +1603,22 @@ cdef FibonacciNode* remove_min(FibonacciHeap* heap):
         unsigned int i
 
     # make all min_node children into root nodes
```

Comment:
If this constructs Binomial Trees, I think it's relevant indicating it (I can't suggest on this line unfortunately).